app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+  has_many :accounts
+  validates :name, presence: true
+  validates :name, uniqueness: true
```

Comment:
The validations on lines 7 and 8 are valid, but need to be placed in the `Account` model

also, you can consolidate to one line (see comment above)